docs/playground/ko/TypeScript/Language/Structural Typing.ts

```diff
@@ -0,0 +1,85 @@
+// TypeScript는 구조적 타입 시스템입니다.
+// 구조적 타입 시스템은 타입을 비교할 때,
+// TypeScript는 타입의 멤버만 고려한다는 것을 의미합니다.
+
+// 두 가지 타입을 만들 수 있지만,
+// 서로 할당할 수 없는 명목적 타입 시스템과는 대조적입니다.
+// 예제를 살펴보세요:nominal-typing
+
+// 예를 들어, 구조적 타입 시스템에서
+// 이러한 두 가지 인터페이스는 완전히 이동시킬 수 있습니다:
+
+interface Ball {
+  diameter: number;
+}
+interface Sphere {
+  diameter: number;
+}
+
+let ball: Ball = { diameter: 10 };
+let sphere: Sphere = { diameter: 20 };
+
+sphere = ball;
+ball = sphere;
+
+// Ball과 Sphere의 모든 멤버를
+// 구조적으로 포함하는 타입을 추가하면,
+// 공이나 구체가 되도록 설정할 수도 있습니다.
```

Comment:
```suggestion
// ball이나 sphere가 되도록 설정할 수도 있습니다.
```
변수 이름도 원문으로 쓰는게 더 잘 이해될 것 같습니다.